blathering-barnacles/Bangazon-2
# Am ticket6

Description: Added the ability to view user Items in the shopping cart and delete them, notice it will actually not delete them but rather add a date on the deletedOn column. The template only displays those which do not have a date.

Steps to Test:
* Click on the shopping cart link.
* You should see the items that belong that the user currently logged in.
* Click on the X of an Item and that item should not be displayed in the shopping cart.

Link to Feature Ticket: [Ticket6](https://github.com/blathering-barnacles/Bangazon-2/issues/6)
